app/sitemap.ts

```diff
@@ -0,0 +1,30 @@
+import { MetadataRoute } from "next";
+
+export default function sitemap(): MetadataRoute.Sitemap {
+  return [
+    {
+      url: "https://0rbit.co/",
+      lastModified: new Date(),
+      changeFrequency: "daily",
+      priority: 1,
+    },
+    {
+      url: "https://0rbit.co/team",
+      lastModified: new Date(),
+      changeFrequency: "daily",
```

Comment:
why the change frequency daily for the team

Reply:
till new ui isnt done let it be daily, indexing will be done more often by the crawlers

Reply:
cool